electron/src/plugins.ts

```diff
@@ -3,7 +3,7 @@ import { mergeWebPlugins, mergeWebPlugin, WebPlugin } from '@capacitor/core';
 
 export * from '@capacitor/core/dist/esm/web/browser';
 export * from '@capacitor/core/dist/esm/web/camera';
-export * from '@capacitor/core/dist/esm/web/clipboard';
```

Comment:
Since you are defining a native Electron plugin, there is no need to export the Web version any longer. This line can be removed. Nice work on the plugin :+1: